### DISCUSSION TITLE
Jean Robert Alves

### DISCUSSION THREAD

**Discussion by jradesenv** [bpm-crafters/bpm-crafters#61] [Request for Hall of Signatories]
My name is Jean Robert Alves, im working mainly as an IT architect for Agibank and started using camunda in 2017. Im always looking to help the community on how to benefit from using BPM in any environment and ofcourse would like to sign the manifesto!

**Comment by Hafflgav** [comment]
Hey @jradesenv 👋 
Thank you for wanting to sign the manifesto. :) 
I think all of us know you - but thank you a lot for the short intro! 

I added you in #62. Once it is being merged your name will also be listed on the Website.
